pkg/buildermgr/pkgwatcher.go

```diff
@@ -100,8 +100,13 @@ func (pkgw *packageWatcher) build(buildCache *cache.Cache, srcpkg *fv1.Package)
 		return
 	}
 
+	// Create a new BackOff for health check on environment builder pod
+	healthCheckBackOff, err := utils.NewBackOff(500*time.Millisecond,time.Minute/10,5)
```

Comment:
So at least for builder pod especially for platforms like JVM the `maxInterval` of 6 seconds seems small. Can we go for something like 120 seconds?